requirements.txt

```diff
@@ -1,3 +1,4 @@
 requests>=2.4.2
 ipaddress; python_version < '3.3'
 PyYAML
+jmespath
```

Comment:
We should not depend on that.

Reply:
I could find the way how to do it without `jmespath`. Suggestions are more than welcomed :)